### PR TITLE
Release 1

### PR DESCRIPTION
**Pull request by schas002**

### CODE REVIEW

File: chatbot.js [hunk 1]
```diff
@@ -80,10 +80,13 @@ var Chatbot = function Chatbot() {
   }, 2500);
 };
 
+var VERSION = 1;
```
Comment (vihanb) [inline chatbot.js]: You should probably make the version a string. So you can do `1.0.1` and other smaller version bumps like: `1.2.113-alpha.14`

Comment (schas002) [inline chatbot.js]: Erm, if the repo releases just after the milestone corresponding to the release finishes, does that count? :/

Comment (schas002) [inline chatbot.js]: But you can do `1.01` for hotfixes.

**Comment by schas002** [comment]
I'll leave two days for doing final refactors and bug fixes, then I'll do testing.

**Comment by schas002** [comment]
Mego disallows me to do the testing, so the release is cancelled for the time being.

@vihanb How can I call the `onmessage` function on an arbitrary string from the JS console? I need this for testing.

**Comment by schas002** [comment]
Version 1 is being released!

**Pull request merged**